app/views/customers/passwords/edit.html.slim

```diff
@@ -0,0 +1,14 @@
+.signup-form-wrapper
+  h2 Change your password
+  = simple_form_for(resource, as: resource_name, url: customer_password_path, html: { method: :put }) do |f|
+    - if resource.errors[:reset_password_token].present?
+      = flash_container(BootstrapFlashHelper::ALERT_TYPES_MAP.fetch(:alert), resource.errors.full_messages.join(', ') )
```

Comment:
Why you use `errors.full_messages` here? As I mentioned before: you check `if resource.errors[:reset_password_token].present?` on the line above, so only if reset_password_token errors present, then you show all errors here... I think there should be only errors related with `reset_password_token`...